print_options.rb

```diff
@@ -0,0 +1,13 @@
+class PrintOptions
+  def show
+    puts ''
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
```

Comment:
- [ ] Great effort working on the refactoring of your project. But currently I notice when I create person (teacher), the age details was not populate when created rather. the figure is populate to the `id`. below is a video demonstration of the what i was talking about

Video 👇 

[screen-capture.webm](https://github.com/SergioPeralta22/MV-School/assets/78634841/54447bfe-5da7-4eb0-a015-e373c8ba80e1)